packages/core-components/sources/faker-demo-source/flights.yaml

```diff
@@ -0,0 +1,35 @@
+rows: 250000
+schema:
+  id:
+    type: id
+  plane:
+    category: airline
+    item: airplane 
+    targetField: name
+  airline:
+    category: airline
+    item: airline 
+    targetField: name
+  fare: 
+    category: number
+    item: float 
+    options:
+      - max: 10000
+      - min: 2
+      - fractionDigits: 2
```

Comment:
If this is supposed to be a single options object the `-` shouldn't be on these lines